src/Directory.Packages.props

```diff
@@ -43,4 +44,4 @@
     <PackageVersion Include="Verify.MsTest" Version="20.1.0" />
     <PackageVersion Include="Verify.DiffPlex" Version="2.2.1" />
   </ItemGroup>
-</Project>
+</Project>
```

Comment:
nit: EOL needed

Reply:
Is that required? I'm use any modifications added by `dotnet format` prior to pushing commits.

Reply:
Github is flagging a warning, hence, I added that comment.

Reply:
fixed